framework/core/instance.cpp

```diff
@@ -158,7 +158,33 @@ Instance::Instance(const std::string &              application_name,
 	std::vector<const char *> active_instance_layers(required_validation_layers);
 
 #ifdef VKB_VALIDATION_LAYERS
-	active_instance_layers.push_back("VK_LAYER_KHRONOS_validation");
+	// Optimal validation layers
+	std::vector<const char *> default_debug_layers{{
```

Comment:
This doesn't build on Android for me

Reply:
Will investigate

Reply:
I wasn't able to get an Android build compiling for reasons that appear unrelated to this PR.